### PR TITLE
Add segformer image classification model on wh_b0 and gs

### PR DESCRIPTION
**Pull request by chandrasekaranpradeep**

### CODE REVIEW

**Comment by nvukobratTT** [comment]
Can you also update:
- model_demos/pyproject.toml
- Main Readme.md file

**Comment by chandrasekaranpradeep** [comment]
Updated the Readme.md file

File: model_demos/README.md [hunk 1]
```diff
@@ -39,10 +39,12 @@ python cv_demos/resnet/pytorch_resnet.py
 | [DeiT](cv_demos/deit/)                                    |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
 | [DenseNet](cv_demos/densenet/)                            |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
 | [DistilBERT](nlp_demos/distilbert/)                       |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
+| [DLA](cv_demos/dla/)                                      |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
```
Comment (nvukobratTT) [inline model_demos/README.md:42]: Instead of version, let's put TBD as we still didn't deliver these models :))

Comment (chandrasekaranpradeep) [reply model_demos/README.md]: Updated the version into TBD

**Pull request merged**